specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2020-10-01/imds.json

```diff
@@ -0,0 +1,910 @@
+          "description": "This is the resource name of the VMSS."
+        },
+        "vmSize": {
+          "type": "string",
+          "description": "This is the size of the VM."
+        },
+        "zone": {
+          "type": "string",
+          "description": "This is the availability zone of the VM."
+        }
+      },
+      "description": "Compute Metadata"
+    },
+    "Network": {
+      "type": "object",
+      "properties": {
+        "interface": {
+          "type": "array",
+          "description": "This contains data about the network interface.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/NetworkInterface"
+          }
+        }
+      },
+      "description": "Network Metadata"
+    },
+    "NetworkInterface": {
+      "type": "object",
+      "description": "This contains data about the network interface.",
+      "properties": {
+        "ipv4": {
+          "type": "object",
+          "description": "This contains the IPv4 address.",
+          "properties": {
+            "ipAddress": {
+              "type": "array",
+              "description": "This is the IP address",
+              "items": {
+                "type": "object",
+                "description": "This contains the IPv4 properties.",
+                "$ref": "#/definitions/Ipv4Properties"
+              }
+            },
+            "subnet": {
+              "type": "array",
+              "description": "This is the subnet",
+              "items": {
+                "type": "object",
+                "description": "This contains the subnet properties.",
+                "$ref": "#/definitions/SubnetProperties"
+              }
+            }
+          }
+        },
+        "ipv6": {
+          "type": "object",
+          "description": "This contains the IPv6 address.",
+          "properties": {
+            "ipAddress": {
+              "type": "array",
+              "description": "This is the IP address",
+              "items": {
+                "type": "object",
+                "description": "This contains the IPv6 properties.",
+                "$ref": "#/definitions/Ipv6Properties"
+              }
+            }
+          }
+        },
+        "macAddress": {
+          "type": "string",
+          "description": "This is the MAC address of the interface."
+        }
+      }
+    },
+    "OsProfile": {
+      "type": "object",
+      "description": "This contains the data about the OS.",
+      "properties": {
+        "adminUsername": {
+          "type": "string",
+          "description": "This is admin account."
+        },
+        "computerName": {
+          "type": "string",
+          "description": "This is the name of the VM."
+        },
+        "disablePasswordAuthentication": {
+          "type": "string",
+          "description": "This specifies whether or not password authentication is disabled. Note that this is present only for Linux VMs."
```

Comment:
What are the possible values here, true/false?

Reply:
True, false, or empty string (in the case of a windows VM where the field is not present in the CRP blob)

Reply:
Might be worth adding this info to the description given the type is string (we generate docs from this info), although it's somewhat implied by the name of the field (I don't know the customer base here well enough to say if they can intuit this).

Reply:
Updated the description to note the empty string value for windows VM.